src/sema/statements.rs

```diff
@@ -1810,6 +1811,17 @@ fn try_catch(
     ns: &mut Namespace,
     diagnostics: &mut Diagnostics,
 ) -> Result<(Statement, bool), ()> {
+    if ns.target == Target::Solana {
+        diagnostics.push(Diagnostic::error(
+            *loc,
+            "The try-catch statement is not supported on Solana. Please, go to \
+             https://solang.readthedocs.io/en/latest/language/statements.html#try-catch-statement \
+             for more information"
```

Comment:
Nitpick, but I'd argue that `for more information` is superfluous.

Reply:
In this case, I believe that if I remove `for more information`, there is not a clear ending to the phrase. Have a look: 
`The try-catch statement is not supported on Solana. Please, go to <site>`